topics/transcriptomics/tutorials/limma-voom/tutorial.md

```diff
@@ -0,0 +1,572 @@
+---
+layout: tutorial_hands_on
+title: Limma-voom (differential expression)
+zenodo_link: "https://figshare.com/s/f5d63d8c265a05618137"
+enable: "false"
+questions:
+  - "What are the expression profiles in the mammary gland of pregnant versus lactating mice?"
+  - "How to analyze RNA count data using limma-voom?"
+objectives:
+  - "Analysis of RNA-seq count data using limma-voom"
+  - "Quality control of count data"
+  - "Visualisation and interactive exploration of count data"
+time_estimation: "3h"
+key_points:
+  - "Use limma-voom tool to perform differential expression and output useful plots"
+  - "Multiple comparisons can be input and compared"
+  - "DE results can be interactively explored with limma-voom via Glimma"
+contributors:
+  - mblue9
+  - Belinda Phipson
```

Comment:
@mblue9 Can you add them to the `CONTRIBUTORS.yaml` file, please? Do they have a GitHub username? If not, add `github: false` tag in the CONTRIBUTORS file